holopy/core/io/io.py

```diff
@@ -30,11 +30,11 @@
 from holopy.core.io import serialize
 from holopy.core.io.image_file_io import load_image, save_image
 from holopy.core.marray import Image, arr_like
-from holopy.core.metadata import Optics
+from holopy.core.metadata import Optics, interpret_args
 from holopy.core.helpers import _ensure_array
 
 
-def load(inf, spacing = None, optics = None, channel=None):
+def load(inf, channel=None, spacing=None):
     """
     Load data or results
 
```

Comment:
I think you may want to delete the spacing and channel arguments here as well, I think basically the same arguments we made with respect to optics apply